api/api.go

```diff
@@ -70,18 +72,39 @@ func New(
 	}, nil
 }
 
-// Register all APIs with the given router and return a mux.
-func (api *API) Register(r *route.Router, routePrefix string) *http.ServeMux {
+// Register all APIs. It registers APIv1 with the provided router directly. As
+// APIv2 works on the http.Handler level, this method also creates a new
+// http.ServeMux and then uses it to register both the provided router (to
+// handle "/") and APIv2 (to handle "<routePrefix>/api/v2"). The method returns
+// the newly created http.ServeMux.
+//
+// If the provided value for timeout is positive, it is enforced for all HTTP
+// requests. (Negative or zero results in no timeout at all.)
+//
+// If the provided value for concurrency is positive, it limits the number of
+// concurrently processed GET requests. Otherwise, the number of concurrently
```

Comment:
See _null_ comment above.

Reply:
Whatever we decide above, we should do the same here, obviously…